examples/chart/teleport-kube-agent/values.schema.json

```
@@ -441,6 +441,35 @@
                 }
             }
         },
+        "serviceAccount": {
+            "$id": "#/properties/serviceAccount",
+            "type": "object",
+            "required": [],
+            "properties": {
+                "name": {
+                    "$id": "#properties/service/name",
+                    "type": "string",
+                    "default": ""
+                },
+                "create": {
+                    "$id": "#properties/service/create",
+                    "type": "boolean",
+                    "default": true
+                }
+            }
+        },
+        "clusterRole": {
+            "$id": "#/properties/clusterRole",
```

Should be `rbac` too

```suggestion
        "rbac": {
            "$id": "#/properties/rbac",
```